Theme/ElegantFin-theme-nightly.css

```diff
@@ -1089,6 +1089,7 @@ div[data-role=controlgroup] a.ui-btn-active {
 .emby-select-withcolor>option {
     color: inherit;
     background-color: var(--selectorBackgroundColor);
+    font-family: Inter,ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji !important;
```

Comment:
I'm using firefox, and it doesn't seem to work without that

Reply:
So I verified that this is only an issue with firefox, so it makes sense to leave it out, as this will likely get fixed with a future update to firefox